index.html

```diff
@@ -0,0 +1,117 @@
+      <a href="index.html" class="img-wrap"><img class="logo" src="./assets/logo.png" alt=""></a>
+      <div class="nav-container">
+        <ul class="links">
+          <li class="link-wrap"><a href="index.html" class="link">Home</a></li>
+          <li class="link-wrap"><a href="about.html" class="link">About</a></li>
+          <li class="link-wrap"><a href="#" class="link">Tickets</a></li>
+          <li class="link-wrap"><a href="#" class="link">Schedule</a></li>
+        </ul>
+      </div>
+      <div class="burger-icon" id="menu-btn">
+        <i class="fa-solid fa-bars"></i>
+        <i class="fa-solid fa-xmark d-none"></i>
+      </div>
+    </nav>
+  </header>
+  <main class="container" id="main-container">
+    <section id="headline">
+      <i class="slogan">"Let the Music Speak!"</i>
+      <h1>RETRO ELECTRIC MUSIC FESTIVAL 2022</h1>
+      <p class="boxed">The most anticipated music festival of all time, reuniting music fans from around the world to hear some of the most popular artists of the moment. You cannot miss it!</p>
+      <strong><time datetime="2022-10-15">2022.10.15(SAT)</time> ~ <time datetime="2022-10-16">16(SUN)</time></strong>
+      <p class="place">@ Grand Oasis Canc&uacute;n, Canc&uacute;n, M&eacute;xico</p>
+    </section>
+    <section id="program">
+      <h2 class="program-title">Main Program</h2>
+      <ul class="program-events">
+        <li class="event">
+          <img src="./assets/program_icon_01.png" alt="" class="icon event-info">
+          <h3 class="name event-info">Conference</h3>
+          <p class="description event-info">Listen to music producers and aritst talk about their way to the top.</p>
+        </li>
+        <li class="event">
+          <img src="./assets/program_icon_02.png" alt="" class="icon event-info">
+          <h3 class="name event-info">Exhibiton</h3>
+          <p class="description event-info">Appreciate the different instruments and merchandising from your favorite idols.</p>
+        </li>
+        <li class="event">
+          <img src="./assets/program_icon_03.png" alt="" class="icon event-info">
+          <h3 class="name event-info">Forum</h3>
+          <p class="description event-info">Have the time to share your thoughts and opinions with all the community online.</p>
+        </li>
+        <li class="event">
+          <img src="./assets/program_icon_04.png" alt="" class="icon event-info">
+          <h3 class="name event-info">Workshop</h3>
+          <p class="description event-info">Try to produce your own music using the different sample some of our featured producers made specially for the event.</p>
+        </li>
+        <li class="event">
+          <img src="./assets/program_icon_05.png" alt="" class="icon event-info">
+          <h3 class="name event-info">Retro Ignite</h3>
+          <p class="description event-info">Get opportunities to network with RE affiliates around the world, also after the festival.</p>
+        </li>
+      </ul>
```

Comment:
The "Main Program" section doesn't look consistent. As you can see in the screenshots, the description isn't looking consistent in all cards. Kindly work on it so that it'll look consistent. 👍🏻 👍🏻 

[Desktop Version]
<img width="1119" alt="Screenshot 2022-07-02 at 10 24 55 PM" src="https://user-images.githubusercontent.com/83048208/177010332-71308add-14cf-4077-9b1d-0d99f72d3106.png">

[Mobile Version]
<img width="379" alt="Screenshot 2022-07-02 at 10 25 19 PM" src="https://user-images.githubusercontent.com/83048208/177010349-3541d8d9-35b3-4279-9554-b5914ab71547.png">